### DISCUSSION TITLE
guide/components/home-box/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#316] [docs-comment]
# guide/components/home-box/

首页布局容器 自定义首页时，使用 <HomeBox> 提供给 区域 的 包装容器。 Props

https://theme-plume.vuejs.press/guide/components/home-box/

**Comment by sunshang-hl** [comment]
可否提供一个自定义首页的示例,比如https://vitepress.dev/zh/ 这样的首页?

**Reply by pengzhanbo** [reply]
这个页面已经都是示例了。像 vitepress 首页的，除了主题能够覆盖到的区域类型，其它的就需要你自定编写组件作为自定义区域类型了。
